_targets.R

```diff
@@ -1,57 +1,12 @@
 library(targets)
-source("1_fetch/src/get_nwis_data.R")
-source("2_process/src/process_and_style.R")
-source("3_visualize/src/plot_timeseries.R")
+source("1_fetch.R")
+source("2_process.R")
+source("3_visualize.R")
 
 options(tidyverse.quiet = TRUE)
 tar_option_set(packages = c("tidyverse", "dataRetrieval")) # Loading tidyverse because we need dplyr, ggplot2, readr, stringr, and purrr
```

Comment:
So what is happening here is by having the `tar_option_set` _after_ your individual phases that are sources is those packages aren't loaded with the individual phases. So all you have to do here is drop the `'source` calls below your `tar_option_set` and then you won't need to load packages within the individual phases.